lib/galaxy/webapps/galaxy/controllers/page.py

```diff
@@ -324,15 +326,32 @@ def create(self, trans, payload=None, **kwd):
         Create a new page.
         """
         if trans.request.method == 'GET':
+            form_title = "Create new Page"
+            title = ""
+            slug = ""
+            content = ""
+            content_format_hide = False
+            content_hide = True
+            if "invocation_id" in kwd:
```

Comment:
I'm okay with this - it is on some level what I was trying to avoid since there isn't an API endpoint for generating the page from the invocation but I suppose it is more or less all staying on the backend minus a quick trip in a hidden textarea. 

I'll approve this PR but I would prefer it if you just stuck the ``invocation_id`` in the form here and then passed it to the page creation API as a parameter - maybe like this ``"content_from": {"src": "workflow_invocation", "id": invocation_id}``. And then did this generation in this if block in the page manager on the backend during page creation.

I think the resulting API would be more usable use non-client uses. Since all the API consumer would need would be the invocation ID and they wouldn't need to know how to use the API that generates the report and they wouldn't need to download it and re-submit it in the request.

Reply:
I understand and I see your point. Thanks for the detailed explanation. I think we might need both, since fully automated generation of a page from the `Invocation ID` might fail if the report contains any type of errors. Those errors are often not caught by the Workflow editor but become apparent after completing the invocation. I chose this strategy since it displays a potential error with an alert and allows the user to modify the imported Markdown content before creating the page.

Reply:
Adding this feature to the page manager such that it becomes available in the regular page api endpoint would look like this: https://github.com/galaxyproject/galaxy/commit/b37fee316ea9123a07a0bd585b72af753766cd7a

Reply:
Excellent point about catching error! I like 89f90d3 also.

Reply:
Great, I added it to this PR and adding a test case for the API now. Thanks for your comments.